### DISCUSSION TITLE
Trouble using the mObywatel app to prove student status. "The image you selected does not appear to contain your school name."

### DISCUSSION THREAD

**Discussion by kapios010** [community/community#75819] [GitHub Education] [answered]
### Hi everyone,

I, like many students in Poland use the mObywatel app for the student ID, which makes it way harder to take a photo. 
Each time I try to apply for the student discount, I get rejected with the reason being that "The image you selected does not appear to contain your school name." while it clearly does.
I suspect that the reason may be that the school name differs in-app from the one chosen on the GitHub Education page, the fact that it's spread out as 2 lines of text or even the fact that I use a laptop webcam, which is not of the best quality. 
If I try to use anything other than the school ID, then it says that I need to use an ID. 
My school will very likely not provide me a paper ID, because it would be not ideal to have both a digital and paper ID at the same time.

I'd appreciate any help with the issue.

**Comment by Ruth678** [comment, marked as answer]
Hi, Kapios
I understand the frustration with the mObywatel app for proving student status. I faced the same issue when working for the [Prime Butcher Wrap](https://www.topcellent.com/butcher-paper-substitute/). Here's a solution:
1) **Check School Name Format:** Ensure the school name on [GitHub matches the app.](https://github.com/topics/match-app) Update if needed.
2) **Improve Image Quality:** Take well-lit photos with a clear view of the entire school name.
3) **Combine School Name:** Capture the name on one line for better recognition.
4) **Contact Support:** Reach out to mObywatel support, explaining the issue and steps taken.
5) **Alternative IDs:** Check if your school provides alternative student IDs for the discount.
I hope these steps help streamline the process. I've been through this, and I understand the frustration. If you need further assistance, feel free to ask.

**Reply by kapios010** [reply]
Only got around to doing it today, but if my response helps any further to anyone who finds this: I entered my school name manually in the textbox (I did not know you could do that) and I stabilized my hand which was holding the phone. And thank you for the response!
